geniza/templates/nav.html

```diff
@@ -42,18 +42,20 @@
                     {% translate "Search" %}
                 </a>
             </li>
-            <li class="menu-item" role="none">
-                <a href="/contact/" role="menuitem">{% translate "Contact Us" %}</a>
-            </li>
+            {% for top_level_page in site.root_page.get_children.live.in_menu %}
+                <li class="menu-item" role="none">
+                    <a href="/{{ top_level_page.localized.slug }}/" role="menuitem">
+                        {{ top_level_page.localized.title }}
+                    </a>
+                </li>
+            {% endfor %}
             <li class="menu-item has-submenu" role="none">
-                {# Translators: label for 'about' link in main navigation #}
-                {% translate "About" as about_label %}
                 <a id="open-about-menu"
                     href="#about-menu"
                     aria-haspopup="true"
                     aria-expanded="false"
                     role="button">
-                    {{ about_label }}
+                    {{ about_page.localized.title }}
                 </a>
                 {# Translators: accessibility label for 'about' menu in navigation #}
                 {% translate "about menu" as about_menu_label %}
```

Comment:
Would like to see this not be hard-coded to the about page set in context, but just iterate over pages and handle a container page with child pages accordingly.

Reply:
Done in 1269ac1 👍